### PR TITLE
SAAS-31 database checks UI MVP

### PR DESCRIPTION
**Pull request by atymchuk**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/798

### CODE REVIEW

**Comment by atymchuk** [comment]
> Looks good, but could be better if add e2e tests and maybe some unit tests for the sanity check

Sure, will do next thing.

File: pmm-app/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "main": "module.js",
   "scripts": {
     "build": "grafana-toolkit plugin:build",
-    "test": "grafana-toolkit plugin:test --updateSnapshot",
+    "test": "grafana-toolkit plugin:test",
```
Comment (lunaticusgreen) [inline pmm-app/package.json:14]: Good fix, but we'll also need command to update snapshots, so please add additional script for it

Comment (atymchuk) [reply pmm-app/package.json:14]: I don't get it: the way it was before, `npm test` would pass even if the snapshots are different. I'm sure that's not the way snapshots have been designed.

Comment (atymchuk) [reply pmm-app/package.json:14]: Anyway, added another script with --updateSnapshot

**Pull request merged**